README.md

```diff
@@ -182,8 +182,8 @@ library will be under the build directory you created.
 ```bash
 # Example on linux after running the build steps above. Assumes the
 # `benchmark` and `build` directories are under the current directory.
-$ g++ -std=c++11 -isystem benchmark/include -Lbuild/src -lpthread \
-  -lbenchmark mybenchmark.cc -o mybenchmark
+$ g++ mybenchmark.cc -std=c++11 -isystem benchmark/include \
+  -Lbenchmark/build/src -lpthread -lbenchmark -o mybenchmark
```

Comment:
I changed the order as you suggested. btw, both ways work on my ubuntu and redhat boxes.

Reply:
it highly depends on how much non-conforming behavior the linker allows.
This might worn with ld.bfd, but i'm pretty sure it won't work with ld.gold/lld